frontend/Home.css

```
@@ -772,6 +772,8 @@
   border-radius: 20px;
   font-size: 28px;
   color: var(--white);
+  transition: transform 0.5s cubic-bezier(0.34, 1.56, 0.64, 1),
```

Stylelint (17.9.0)

[error] Duplicate property "transition" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)